lib/Elevate/Components/MySQL.pm

```diff
@@ -65,6 +63,12 @@ sub _reinstall_mysql_packages {
 
     my ( $major, $minor ) = split( /\./, $mysql_version );
 
+    # Try to restore any .rpmsave'd configs before we reinstall
```

Comment:
should we consider any `.rpmsave` file? what if that `.rpmsave` is several months/years old? is it still a good idea?

Reply:
Hmm, well in this case I very much doubt it will be anything *other* than the my.cnf that we just uninstalled earlier in the elevation process.

That said, it may be worth testing to put a broken one in place before elevation, then seeing if the broken one is restored (RPM in this case refusing to overwrite the old rpmsave'd one would be the only thing to lead to this). I'll leave a note to the tester about this.

Reply:
Alternatively we can check for the presence of the file pre-elevate, check it's timestamp and ignore the restore if this is the same one...